capitalize_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class CapitalizeDecorator < Decorator
```

Comment:
- [ ] Kindly import the `Decorator` class before inheriting it to avoid any error.